docs/localization.md

```diff
@@ -0,0 +1,143 @@
+# Localization 
+
+You can localize your Catalyst storefront so the theme appears in the shopper's preferred language.
+Your storefront displays the language stored in the shopper's browser cookie if you provided translations for that language. 
+Shoppers can then see theme content in the language of their choice throughout their browsing and checkout experience.
+This provides a personalized shopping experience when you sell products internationally. 
+
+Catalyst uses [Next.js App Router Internationalization (i18n)](https://next-intl-docs.vercel.app/docs/getting-started/app-router) library to handle localization.
+
+This guide describes how to provide static string translations for shoppers, including the required subdirectory, file structure, and browser settings. 
+
+[!Note]
+> - Each storefront can only support a single language. To display multiple languages, we recommend setting up a separate store for each language.
+> - To fully localize a store for a language or region, you will need to customize Storefront content in the BigCommerce control panel. Catalyst does not translate content rendered from a store's catalog database (for example, the name of a product). 
+
+
+## Required subdirectory
+
+Each Catalyst project reserves a top level `/messages/` subdirectory for localization.
+For your theme to function properly, both the `/messages/` subdirectory and the `/messages/en.json` file, which contains the English-language default, must be present.
+You can localize your theme by providing other JSON translation files in the `/messages/` directory. 
+
+[!Note]
+> #### Default language
+> Catalyst hardcodes the `en.json` file as the default language file, which contains U.S. English translations when you set up the project [using the CLI](https://www.catalyst.dev/docs/cli).
+
+## Translation file
+
+To translate your theme, create a JSON file for each language you choose to support. Each language that you want to support must have its own JSON file. These can include non-U.S. versions of English, each with their own spellings.
+
+Name your translation files based on the [BCP 47 specification](https://tools.ietf.org/html/bcp47) of language tags and region codes. For an overview of how these language tags are constructed, see [Language tags in HTML and XML](http://www.w3.org/International/articles/language-tags/).
```

Comment:
Is this section accurate? I took it off the Stencil docs

Reply:
In my opinion, it's accurate.